doc/nrf/dm_ncs_distro.rst

```diff
@@ -0,0 +1,64 @@
+**********************************
+
+The simplest way to become part of the |NCS| is to be included in its official `west manifest file`_.
+This way, your own :ref:`module repository <zephyr:modules>` will automatically be part of the |NCS| official distribution and will be included in all releases.
+
+This solution is reserved for selected vendors and requires a commercial agreement with Nordic Semiconductor.
+Contact your local representative if you wish to be considered for inclusion.
+
+Some examples of this type of integration are:
+
+* :ref:`Memfault <mod_memfault>`.
+  This functionality is present through the addition of a `west project in the nRF Connect SDK manifest <Memfault entry in the manifest_>`_.
+  The `repository provided by Memfault <Memfault firmware SDK_>`_ is public.
+  By default, |NCS| users will obtain a copy of it (as well as the integration code necessary for |NCS| based applications) when :ref:`getting the code <cloning_the_repositories>`.
+
+* ANT protocol support.
+  The code and documentation of this protocol is confidential, and access to the private module repository requires the GitHub user to be added to it.
+  This is why the west project is disabled by default, using the :ref:`project groups <zephyr:west-manifest-groups>` feature of west.
+  This means that the ANT entry in the manifest is disabled by default through its presence in the `manifest group filter`_.
+  You will not get a copy of this repository when :ref:`getting the code <cloning_the_repositories>`, and instead, need to enable the repository first using west itself to fetch it locally::
+
+    west config manifest.group-filter +ant
+    west update
+
+Providing a Zephyr module
+*************************
+
+This approach is essentially identical to the previous one.
+However, the :ref:`module repository <zephyr:modules>` you create will not be part of the |NCS| official distribution by default.
+Instead, you will need to host your own documentation explaining how to add the module you make available to user's :ref:`own manifest <dm_workflow_4>`.
+
+An example of this approach is the `Golioth Zephyr SDK`_, which is set up as a `Zephyr module <Golioth module.yml_>`_ that a user can add to their own manifest.
+
+Providing a west manifest
+*************************
+
+Instead of just providing a module repository like in the previous approach, you can also include a west manifest that imports the |NCS| in it, see :ref:`dm_workflow_4`.
+With this approach, you can import your manifest directly.
+It will then in turn import the |NCS| `west manifest file`_.
+
+Just like in the previous approach, Golioth provides an example of such a west manifest, `west-ncs.yml <Golioth west-ncs.yml_>`_ along with a `guide document <Golioth README nRF Connect SDK_>`_.
+
+Forking the |NCS|
+*****************
+
+You can always :ref:`fork all or parts of the nRF Connect SDK <dm-wf-fork>` and then make all the changes required by your features.
```

Comment:
Had to do this in the past to change the mcuboot module, needed both mcuboot and sdk-nrf forked (as if you just tried to skip getting mcuboot in a top level west manifest and added your own, a fork of sdk-mcuboot, it did not work) - is it worth mentioning something like this an an example of when a fork would be needed?

Reply:
I think this would be going into too much detail for the scope of this section. Instead this info would belong somewhere like here:
https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/dm_adding_code.html#workflow-3-application-in-a-fork-of-sdk-nrf